src/net/sourceforge/kolmafia/svn/SVNManager.java

```diff
@@ -515,8 +515,7 @@ private static boolean validateRepo(SVNURL repo, boolean quiet) {
 
     if (failed && !quiet) {
       KoLmafia.updateDisplay(
-          MafiaState.ERROR,
-          "The requested repo failed validation.  Complain to the script's author.");
+          MafiaState.ERROR, "The requested repo (" + repo.getPath() + " failed validation.");
```

Comment:
@jaadams5  This is missing a closing bracket

Reply:
@gausie 
Would you recheck, please?  On my system the only diff is in the string being passed to updateDisplay.

If failed has a an opening bracket, the else has the closing and then an opening and then there is a close before the return.

Reply:
I think he means in the error text.

+          MafiaState.ERROR, "The requested repo (" + repo.getPath() + "**)** failed validation.");

Reply:
See https://github.com/kolmafia/kolmafia/pull/73

I tend to use "bracket" for { or [ and "parenthesis" for ( so I was looking for the wrong thing.